### PR TITLE
Me/dpc 3762 update post

### PR DESCRIPTION
**Pull request by MEspositoE14s**
## 🎫 Ticket

https://jira.cms.gov/browse/DPC-3762

## 🛠 Changes

1. Ip address in POST cannot be null.
2. Ip address is now taken in as String, not an object.
3. Ip address parameter is now taken in as `ip_address`.

## ℹ️ Context for reviewers

See https://github.com/CMSgov/dpc-app/pull/2016#discussion_r1447720721, https://github.com/CMSgov/dpc-app/pull/2016#discussion_r1447721646 and https://github.com/CMSgov/dpc-app/pull/2016#discussion_r1447723812

## ✅ Acceptance Validation

(How were the changes verified? Did you fully test the acceptance criteria in the ticket? Provide reproducible testing instructions and screenshots if applicable.)

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

**Review by ashley-weaver — APPROVED** [review]
👍

File: dpc-api/src/main/java/gov/cms/dpc/api/models/CreateIpAddressRequest.java [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
-import gov.cms.dpc.api.converters.InetDeserializer;
-import io.hypersistence.utils.hibernate.type.basic.Inet;
+import javax.validation.constraints.NotNull;
 
 public class CreateIpAddressRequest {
     private String label;
-    @JsonDeserialize(using = InetDeserializer.class)
-    private Inet ipAddress;
+
+    @NotNull
```
Comment (kyeah) [inline dpc-api/src/main/java/gov/cms/dpc/api/models/CreateIpAddressRequest.java:9]: Very minor, but `@NotEmpty` for string so `""` doesn't get past the validation 😄

**Review by kyeah — APPROVED** [review]
Thanks! 

I think this will still allow invalid IP addresses to get through (until it reaches the database and returns a 500 error) but I realize this isn't covered in the ticket A/C so we can create a followup ticket and/or PR for that.

File: dpc-api/src/main/java/gov/cms/dpc/api/models/CreateIpAddressRequest.java (hunk 1 above)
Comment (MEspositoE14s) [reply dpc-api/src/main/java/gov/cms/dpc/api/models/CreateIpAddressRequest.java]: Just updated.

**Comment by ashley-weaver** [comment]
> I think this will still allow invalid IP addresses to get through...

The frontend code checks for validity, FWIW

**Comment by MEspositoE14s** [comment]
> Thanks!
> 
> I think this will still allow invalid IP addresses to get through (until it reaches the database and returns a 500 error) but I realize this isn't covered in the ticket A/C so we can create a followup ticket and/or PR for that.

I just added a quick check in the IpAddressResource so we'll catch things a little earlier.

**Pull request merged**